### PR TITLE
feat(image-make): script and document to release docker image

### PR DESCRIPTION
**Pull request by definite**
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-docker-files/15)

### CODE REVIEW

File: image-make [hunk 1]
```diff
@@ -131,18 +131,18 @@ for tag in "${TagArray[@]}";do
...
     fi
     set -x
     docker tag $BuiltImage $ZANATA_DOCKER_PUSH_REGISTRY/$Image
     set +x
     if [ $IsPush -eq 1 ];then
-	set -x
-	docker push $ZANATA_DOCKER_PUSH_REGISTRY/$Image
-	set +x
+        set -x
+        docker push $ZANATA_DOCKER_PUSH_REGISTRY/$Image
```
Comment (carlosmunoz) [inline image-make:147]: what about authentication against the Docker registry? We might need to either mention that needs to be done beforehand, or include it somehow in the script

File: README.md [hunk 2]
```diff
@@ -1,6 +1,34 @@
 # Zanata Docker Images
 
-Docker images for Zanata.
+Docker images for Zanata project.
+
+## Release docker image
```
Comment (carlosmunoz) [inline README.md]: Do we really need a script for this?

Comment (definite) [reply README.md]: We need to run 5 commands (one build, tag for version and latest,  and push for version and latest). Using this script, I no longer forget to tag and push version and latest.

**Comment by definite** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[image-make, line 144 at r1](https://reviewable.io:443/reviews/zanata/zanata-docker-files/15#-Kap5fyjxv-r3xRTatXs-r1-144:-KawJ39KJgKZCIuPhYLo:b-896fix) ([raw file](https://github.com/zanata/zanata-docker-files/blob/efd36805deb01132147b4c3ffb77bea9aaef4a5f/image-make#L144)):*
<details><summary><i>Previously, carlosmunoz (Carlos Munoz) wrote…</i></summary><blockquote>

what about authentication against the Docker registry? We might need to either mention that needs to be done beforehand, or include it somehow in the script
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-docker-files/15)*

**Comment by carlosmunoz** [comment]
Review status: 0 of 2 files reviewed at latest revision, 3 unresolved discussions.

---

*[image-make, line 13 at r2](https://reviewable.io:443/reviews/zanata/zanata-docker-files/15#-Kb7UQDhF68K7HwdIcPX:-Kb7UQDhF68K7HwdIcPY:b9fr9jc) ([raw file](https://github.com/zanata/zanata-docker-files/blob/739d35dae75299425252a3a3fbaead386a7afb52/image-make#L13)):*
> ```
> ### Note that this program assumes you are already have an docker registry
> ### account and login in from the command line at your workstation.
> ### Refer README.md for instruction.
> ```

Some corrections / suggestions:

"Note that this _script_ ~~program~~ assumes you ~~are~~ _already have_ ~~an~~ a docker registry account and ~~login~~ _have logged_ in from the command line at your workstation."

---

*[README.md, line 5 at r1](https://reviewable.io:443/reviews/zanata/zanata-docker-files/15#-Kap5fyjxv-r3xRTatXr-r1-5:-Kb7VMuO3E_frS_O9sfd:b-odkata) ([raw file](https://github.com/zanata/zanata-docker-files/blob/efd36805deb01132147b4c3ffb77bea9aaef4a5f/README.md#L5)):*
<details><summary><i>Previously, definite (Ding-Yi Chen) wrote…</i></summary><blockquote>

We need to run 5 commands (one build, tag for version and latest,  and push for version and latest). Using this script, I no longer forget to tag and push version and latest.
</blockquote></details>

Ok, then perhaps the script should be titled something like `image-release` as it not only builds it, but goes beyond that.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-docker-files/15)*

**Comment by definite** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[image-make, line 13 at r2](https://reviewable.io:443/reviews/zanata/zanata-docker-files/15#-Kb7UQDhF68K7HwdIcPX:-Kb7gIQ3nZWVFNjbE5xc:b-896fix) ([raw file](https://github.com/zanata/zanata-docker-files/blob/739d35dae75299425252a3a3fbaead386a7afb52/image-make#L13)):*
<details><summary><i>Previously, carlosmunoz (Carlos Munoz) wrote…</i></summary><blockquote>

Some corrections / suggestions:

"Note that this _script_ ~~program~~ assumes you ~~are~~ _already have_ ~~an~~ a docker registry account and ~~login~~ _have logged_ in from the command line at your workstation."
</blockquote></details>

Done.

---

*[README.md, line 5 at r1](https://reviewable.io:443/reviews/zanata/zanata-docker-files/15#-Kap5fyjxv-r3xRTatXr-r1-5:-Kb7gK25gVgAMJYTr2bQ:b-9lpgct) ([raw file](https://github.com/zanata/zanata-docker-files/blob/efd36805deb01132147b4c3ffb77bea9aaef4a5f/README.md#L5)):*
<details><summary><i>Previously, carlosmunoz (Carlos Munoz) wrote…</i></summary><blockquote>

Ok, then perhaps the script should be titled something like `image-release` as it not only builds it, but goes beyond that.
</blockquote></details>

As discussed, the name of the script is kept.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-docker-files/15)*

**Comment by carlosmunoz** [comment]
Reviewed 1 of 2 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-docker-files/15)*

**Comment by definite** [comment]
http://stackoverflow.com/a/31334443/350580

**Comment by carlosmunoz** [comment]
Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-docker-files/15)*

**Comment by djansen-redhat** [comment]
✅ This seems to make my SELinux very cranky, but it otherwise works

**Pull request merged**